great_expectations/checkpoint/actions.py

```diff
@@ -1023,3 +1031,47 @@ def _run(
             data_docs_validation_results[sites["site_name"]] = sites["site_url"]
 
         return data_docs_validation_results
+
+
+class CloudNotificationAction(ValidationAction):
+    """
+    CloudNotificationAction is an action which utilizes the Cloud store backend
+    to deliver user-specified Notification Actions.
+    """
+
+    def __init__(
+        self,
+        data_context: "DataContext",
+        checkpoint_ge_cloud_id: str,
+    ):
+        super().__init__(data_context)
+        self.checkpoint_ge_cloud_id = checkpoint_ge_cloud_id
+
+    def _run(
+        self,
+        validation_result_suite: ExpectationSuiteValidationResult,
+        validation_result_suite_identifier: GeCloudIdentifier,
+        data_asset=None,
+        payload: Optional[Dict] = None,
+        expectation_suite_identifier=None,
+        checkpoint_identifier=None,
+    ):
+        logger.debug("CloudNotificationAction.run")
+
+        if validation_result_suite is None:
+            return
```

Comment:
Is a silent return appropriate here? perhaps a log for skipping action?

Reply:
seconded ☝🏼

Reply:
Good point! I was just following previously established pattern, but it would be nice to have a log. Adding.